libvips/arithmetic/math.c

```diff
@@ -164,6 +164,12 @@ vips_math_buffer( VipsArithmetic *arithmetic,
 	case VIPS_OPERATION_MATH_ASIN: 	SWITCH( ADSIN ); break;
 	case VIPS_OPERATION_MATH_ACOS: 	SWITCH( ADCOS ); break;
 	case VIPS_OPERATION_MATH_ATAN: 	SWITCH( ADTAN ); break;
+	case VIPS_OPERATION_MATH_SINH: 	SWITCH( sinh ); break;
```

Comment:
Like almost all image processing packages, libvips uses degrees everywhere, so I think it'd be very confusing to have these functions in radians. I would add:

```
#define DSINH( X ) (sinh( VIPS_RAD( X ) ))
#define DCOSH( X ) (cosh( VIPS_RAD( X ) ))
#define DTANH( X ) (tanh( VIPS_RAD( X ) ))
```

And the same for asinh() etc.

Reply:
I've never heard of wanting to calculate hyperbolic functions of an angle in degrees (even though they're related to the trig functions by i).  For example the numpy docs specify that [`sin`](https://numpy.org/doc/stable/reference/generated/numpy.sin.html) uses radians, but don't even bother saying anything for [`sinh`](https://numpy.org/doc/stable/reference/generated/numpy.sinh.html).  The calculator on Windows has a switch for radians or degrees for the trig functions, but it doesn't affect the hyperbolic functions.

On the other hand the [C# docs](https://docs.microsoft.com/en-us/dotnet/api/system.math.sinh?view=net-5.0) do say that it's in radians, and I see a few [examples](https://docs.trifacta.com/display/SS/COSH+Function) of calculating hyperbolic functions of degree angles.

I see your point.  Personally I still think it's less confusing to keep the hyperbolic functions in radians, but I can switch it to degrees if you want me to.

Reply:
You're right, I checked my elderly HP calculator and the rad/deg mode does not affect the hyperbolic functions. That's good enough for me -- let's leave this has you had it originally.

Could you add a note to the docs below saying these functions use radians?

Reply:
(sorry for wasting your time)